analysis/plotting.py

```diff
@@ -179,7 +179,8 @@ def write_coverage_growth_plot(self, benchmark_df, image_path, wide=False):
         self._write_plot_to_image(self.coverage_growth_plot,
                                   benchmark_df,
                                   image_path,
-                                  wide=wide)
+                                  wide=wide,
+                                  logscale=False)
```

Comment:
This seems wrong to me, i think you need to add logscale argument and then do logscale=logscale here.

Reply:
yep, fixed this another related issue